content/en/includes/task-tutorial-prereqs.md

```diff
@@ -1,5 +1,5 @@
-You need to have a Kubernetes cluster, and the kubectl command-line tool must
-be configured to communicate with your cluster. It is recommended to run this tutorial on a cluster with at least two nodes that are not acting as control plane hosts. If you do not already have a
+The node must be running Linux. you need to have a Kubernetes cluster, and the kubectl command-line
```

Comment:
```suggestion
You need to have a Kubernetes cluster, and the kubectl command-line
```

Reply:
A more descriptive statement in the form of a note before this statement would make more sense. As it stands currently, there is no mention of sysctl anywhere.  Merely adding the statement you have does not help clarify that sysctl doesn't do anything for Windows nodes. WDYT?

Reply:
Hi @divya-mohan0209 thanks for suggesting I agree that keeping this statement more descriptive in the form of a Note. will look more precise for a reader. I will add those changes